README.md

```diff
@@ -21,17 +21,69 @@ dependencies, you need to require
 }
 ```
 
-Please, read the website to [get more informations about how to
-install](http://hoa-project.net/Source.html).
+And add `RulerBundle` to your `AppKernel`:
+
+```php
+// app/AppKernel.php
+
+class AppKernel extends Kernel
+{
+    // …
+
+    public function registerBundles()
+    {
+        $bundles = array(
+            // …
+            new Hoathis\Bundle\RulerBundle\HoathisRulerBundle(),
+        );
+
+        // …
+    }
+}
+```
 
 ## Quick usage
 
-…
+### Ruler service
 
-## Documentation
+The `RulerBundle` defines a `hoathis.ruler` service that can be used as
+described in [Ruler's documentation](http://hoa-project.net/En/Literature/Hack/Ruler.html).
```

Comment:
Please, remove `/En` from the URL, to automatically redirect the user to its own language.
Also use “`Hoa\Ruler`” (with `<code>`) instead of “Ruler”.